Rakefile

```diff
@@ -1,29 +1,32 @@
-SWIFTLINT_VERSION='0.27.0'.freeze
-XCODE_WORKSPACE='WordPress.xcworkspace'.freeze
-XCODE_SCHEME='WordPress'.freeze
-XCODE_CONFIGURATION='Debug'.freeze
+# frozen_string_literal: true
+
+require 'English'
```

Comment:
TIL about the [English library](https://ruby-doc.org/stdlib-2.7.6/libdoc/English/rdoc/English.html)

> Include the English library file in a Ruby script, and you can reference the global variables such as VAR{$_} using less cryptic names